dao/submission.js

```diff
@@ -226,11 +226,9 @@ class SubmissionDAO extends GenericDAO {
             const studyScope = userScope.getStudyScope();
             // If not assigned all studies then add assigned studies filters
             if (!isAllStudy(studyScope?.scopeValues)) {
+                const studyScope = userScope.getStudyScope();
                 const studyIDs = studyScope?.scopeValues || [];
-                // Only add studyID filter if there are actual study IDs to filter by
-                if (studyIDs.length > 0) {
```

Comment:
if we add this studyIDs.length > 0, it will show all submission when the study scope values is empty.